nav2_dwb_controller/dwb_core/src/dwb_local_planner.cpp

```diff
@@ -89,6 +89,9 @@ void DWBLocalPlanner::configure(
     rclcpp::ParameterValue(true));
   declare_parameter_if_not_declared(
     node, dwb_plugin_name_ + ".prune_distance",
+    rclcpp::ParameterValue(1.0));
```

Comment:
Make sure the behavior is the same as before, so the prune / forward prune distance default parameters are the same as before

Reply:
Ok, done.